fabbwled-frontend/src/routes/game/Game.tsx

```diff
@@ -1,51 +1,139 @@
 import styles from "./Game.module.scss";
+import { useEffect, useState } from "react";
+import DiceComponent from "../../DiceComponent.tsx";
+import { convertToElement } from "../../SectionRenderer.tsx";
+import { Player, Section } from "../../interfaces/character.ts";
 
 export default function Game() {
+  const [character, setCharacter] = useState<Player>();
+  const [section, setSection] = useState<Section>();
+
+  useEffect(() => {
+    getCharacter().then((player: Player) => {
+      setCharacter(player);
+      getSection(player.currentSection.split("-")[1]).then(setSection);
+    });
+  }, []);
+
+  const getCharacter = async () => {
+    const response = await fetch("http://localhost:8080/api/player");
+    return response.json();
+  };
+
+  console.log(character);
+  function rollDice(diceNumber: number) {
+    console.log(diceNumber);
+  }
+
+  function newGame() {
+    document.cookie = "";
+  }
+
+  function exitGame() {
+    document.cookie = "";
+    window.location.href = "about:blank";
+  }
+
+  const clickItem = (clickId: number) => {
+    console.log(clickId);
+    return fetch(`http://localhost:8080/api/section/click`, {
+      method: "POST",
+      body: JSON.stringify({
+        clickId: clickId,
+      }),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    }).then((res) => res.json());
+  };
+
+  const getSection = async (sectionId: string): Promise<Section> => {
+    try {
+      const response = await fetch(
+        `http://localhost:8080/api/section/1/${sectionId}`,
+      ); //${character?.currentSection.sectionId}
+      const data = await response.json();
+      console.log(data);
+      return data;
+    } catch (e) {
+      return {
+        body: {
+          type: "CONTAINER",
+          style: "NONE",
+          children: [
+            {
+              type: "SIMPLE",
+              text: "Haha existiert nicht",
```

Comment:
This makes sense for a 404, but I don't think it makes sense for a network error, it should say something about network errors